builtin/fstk/ui.lua

```diff
@@ -61,7 +61,17 @@ function ui.update()
 	local formspec = {}
 
 	-- handle errors
-	if gamedata ~= nil and gamedata.reconnect_requested then
+	if gamedata ~= nil and gamedata.reconnect_requested and #gamedata.transfer_address > 0 then
```

Comment:
`#gamedata.transfer_address > 0` would be slightly more readable as `gamedata.transfer_address ~= ""` (also: why not use `nil` to indicate an absent transfer address?)